docs/articles/advanced_topics/generic_host.md

```diff
@@ -0,0 +1,141 @@
+---
+uid: articles.advanced_topics.generic_host
+title: Generic Host
+---
+
+# Introduction
+A .NET Generic Host is a reusable, lightweight, and extensible hosting framework that provides a consistent way to host different types of .NET applications. It is designed to simplify the startup process and provide a common infrastructure for configuring, logging, dependency injection, and other common tasks required by modern applications.
+
+It allows developers to build console applications, background services, and other types of .NET applications that can run as standalone processes, Windows services, or Docker containers, among other deployment options. By using a generic host, developers can focus on implementing the core business logic of their application rather than dealing with the infrastructure and plumbing required to host and manage it.
+
+You can read more about Generic hosts [here](https://learn.microsoft.com/en-us/dotnet/core/extensions/generic-host).
+
+# Making a Bot using the Generic Host
+## Setting up the Builder
+To get started, you'll need to write some code that configures and runs your application. Here's a simple example that shows how to use the Generic Host to run a bot service:
+```cs
+private static async Task Main()
+{
+    await Host.CreateDefaultBuilder()
+      .UseConsoleLifetime()
+      .ConfigureServices((hostContext, services) =>
+      {
+          services.AddHostedService<YourBotHost>();
+      })
+      .RunConsoleAsync();
+}
+```
+This code does a few things. First, it calls the `Host.CreateDefaultBuilder()` method, which creates a new `IHostBuilder` instance with some default settings. Then, it calls the `UseConsoleLifetime()` method to configure the lifetime of the host. This tells the host to keep running until it receives a [SIGINT Signal](https://en.wikipedia.org/wiki/Signal_(IPC)#SIGINT) i.e. when the user presses Ctrl+C in the console, or when another program tells it to stop.
```

Comment:
Are we certain that it listens only for the `SIGINT` signal and not any of the other POSIX signals as well?